x/datapool/types/keys.go

```diff
@@ -59,6 +59,7 @@ func GetKeyPrefixNFTRedeemReceiptByPoolID(poolID uint64, redeemer sdk.AccAddress
 	return append(keyPoolAppended, redeemer.Bytes()...)
 }
 
-func GetKeyPrefixNFTRedeemReceipt(poolID, round uint64, redeemer sdk.AccAddress) []byte {
-	return append(GetKeyPrefixNFTRedeemReceiptByPoolID(poolID, redeemer), sdk.Uint64ToBigEndian(round)...)
+func GetKeyPrefixNFTRedeemReceipt(poolID, round, nftID uint64, redeemer sdk.AccAddress) []byte {
+	keyRoundAppended := append(GetKeyPrefixNFTRedeemReceiptByPoolID(poolID, redeemer), sdk.Uint64ToBigEndian(round)...)
+	return append(keyRoundAppended, sdk.Uint64ToBigEndian(nftID)...)
```

Comment:
Hmm.. I think I need to use `combineKeys` function as I did in `datadeal`.

Reply:
Yes, maybe it is necessary

Reply:
I applied `CombineKeys` functions in [3dadde8](https://github.com/medibloc/panacea-core/pull/322/commits/3dadde81fdbd949820948806b7083115481c36c5). But I'm not sure it's right.

Reply:
I think that datapool's own CombineKey function is needed.
I have the same need, so I made one.

https://github.com/medibloc/panacea-core/pull/318/files#diff-ac95601e1086d36dbd2dfc79234fb1e3e65b741a550d897afe1c1215cd316849R73

Reply:
In the end, I plan to change this.
https://github.com/medibloc/panacea-core/blob/ft/278/curator_commission_rate/x/datapool/types/keys.go#L73